src/components/modal/styles.ts

```diff
@@ -31,7 +31,8 @@ const showTranslateY = keyframes`
 
 export const ModalBox = styled(Box)<ModalBoxProps>`
   min-width: ${({ modalMinWidth }) => modalMinWidth || "32rem"};
-  overflow: hidden;
+  max-width: 95vw;
+  overflow: scroll;
```

Comment:
I think we want "auto" here.

scroll
Content is clipped if necessary to fit the padding box. Browsers always display scrollbars whether or not any content is actually clipped, preventing scrollbars from appearing or disappearing as content changes. Printers may still print overflowing content.

auto
Depends on the [user agent](https://developer.mozilla.org/en-US/docs/Glossary/User_agent). If content fits inside the padding box, it looks the same as visible, but still establishes a new block formatting context. Desktop browsers provide scrollbars if content overflows.

Reply:
Yes, right, `auto` will be better 👍